src/types/chat.rs

```diff
@@ -28,6 +28,12 @@ pub struct Chat {
     ///
     /// [`GetChat`]: crate::payloads::GetChat
     pub pinned_message: Option<Box<Message>>,
+
+    /// The time after which all messages sent to the chat will be automatically
+    /// deleted; in seconds. Returned only in [`GetChat`].
```

Comment:
```suggestion
    /// The time after which all messages sent to the chat will be automatically
    /// deleted; in seconds. Returned only from [`GetChat`].
```

Reply:
This is from the [official documentation](https://core.telegram.org/bots/api#chat):
![image](https://user-images.githubusercontent.com/38225716/117028229-2cebec00-ad06-11eb-90bf-efd5c72ca50e.png)